projects/plugins/jetpack/_inc/client/components/money-back-guarantee/icons.jsx

```diff
@@ -0,0 +1,15 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+export const yearly = (
```

Comment:
I think we just need a more descriptive name here and we are good.

Also, if you want to take advantage of how React works, we can turn this into a component, then we'll have access to props and can set custom `class`es, `title` and etc. But this is just me overthinking it, this might not be needed at all - though, I'd like to read a more descriptive name anyway.

Reply:
The reason it's called `yearly` is because the icon includes the number `14` which means "14 days back guarantee", so I actually like the name.
If we ever want to show monthly versions of products along with this component we would have to introduce a `monthly` icon as well.

If you look at the commit history you'll see that I've already been down the road of making this much "smarter" but I generally like to keep it as simple as possible and since this is a static value (for now, at least), this just made sense 😅 
I also just used the `svg` as a background and showed a dynamic number on top but because the icon is shown in a small size, the dynamic number was never truly centered across all browsers. This implementation was just the easy solution to the number always being centered and looking more crisp 🙈 

Honestly: I don't necessarily want to spent too much time in making this much smarter since the UI generally is going to be moved towards using Jetpack RNA components which means new grid, font, color etc., so this is more of an "in between original and RNA components" solution.